runtime/src/main/java/org/corfudb/runtime/view/TableRegistry.java

```diff
@@ -458,19 +458,23 @@ public static UUID getStreamIdForStreamTag(String namespace, String streamTag) {
         List<String> streamTagsStringList = defaultValueMessage.getDescriptorForType()
                 .getOptions().getExtension(CorfuOptions.tableSchema).getStreamTagList();
 
-        Set<UUID> streamTagsUUIDForTable = streamTagsStringList
+        Map<UUID, String> streamTagsForTable = streamTagsStringList
```

Comment:
This part of the code seems a bit messy. Can we add more type safety and clarity?
My proposal is:
1. add a small `StreamInfo` value class into the ObjectView class
```
    @AllArgsConstructor
    @Getter
    public static class StreamInfo {
        private final String name;
        private final UUID id;
    }

 ...
public static final StreamInfo LOG_REPLICATOR_STREAM_ID = new StreamInfo(
            "Transaction_Stream",
            CorfuRuntime.getStreamID("Transaction_Stream")
    );
```

2. make current code cleaner:
```
Stream<StreamInfo> streamTags = streamTagsStringList
                .stream()
                .map(tag -> new StreamInfo(tag, getStreamIdForStreamTag(namespace, tag)));

        if (isFederated) {
            Stream<StreamInfo> logReplicatorStreamId = Stream.of(ObjectsView.LOG_REPLICATOR_STREAM_ID);
            streamTags = Stream.concat(logReplicatorStreamId, streamTags);
        }

        Set<String> streamTagsUUIDForTable  = streamTags
                .map(StreamInfo::getName)
                .collect(Collectors.toSet());
```

note: if you'd like to touse my approach let's have a chat

Reply:
BTW, is it ok that there is no namespace for LOG_REPLICATOR_STREAM_ID?

Reply:
And then we can simplify it more:
```
Set<UUID> streamTags = defaultValueMessage
                .getDescriptorForType()
                .getOptions()
                .getExtension(CorfuOptions.tableSchema)
                .getStreamTagList()
                .stream()
                .map(tag -> new StreamInfo(tag, getStreamIdForStreamTag(namespace, tag)))
                .map(StreamInfo::getId)
                .collect(Collectors.toSet());

        if (isFederated) {
            streamTags.add(ObjectsView.LOG_REPLICATOR_STREAM_ID.getId());
        }
```

Reply:
If we simplify in this manner, how do we log the stream tag names along with their corresponding UUIDs?

Reply:
Then the previous one should work

Reply:
I thought if this change is only for improving client side logging, i.e. the log in line 476 if I understood correctly, then the inline manner as @zfrenette initially made is fine. 

But yeah the `static class StreamInfo` approach would make it cleaner and more extendable if we will go deeper into the stream name logging. Might be good to consider it together with the server side logging change like @hisundar previously made in #2978 ?